### PR TITLE
Create team page

### PR DESCRIPTION
**Pull request by DerrickPersaud**
Finished team page, missing functionality to delete members and delete pending requests 
Also you can currently add a member and pending email more than once 

Screenshot of an owner nest view 
image

Screenshot of a nest you're in but you're not the owner
image

### CODE REVIEW

**Comment by DerrickPersaud** [comment]
To do in future commits 
1. Change the route for team page 
2. Delete functionality (delete member or pending)

File: src/App.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ class App extends Component {
     this.setState({
       username,
     });
+
+    Auth.currentUserInfo().then((value) => console.log("user info is:", value));
```
Comment (brianseidl) [inline src/App.js]: I think we can remove this right

File: src/index.css [hunk 2]
```diff
@@ -287,3 +367,25 @@ li a.dropdown-toggle::after {
...
 
 /* End Board Classes */
+
+/* Generic Form Controls */
+.disabled-field {
+  pointer-events: none;
+  opacity: 0.6;
+}
+
+/* button[disabled]:active,
```
Comment (brianseidl) [inline src/index.css]: remove unused CSS

**Review by brianseidl — CHANGES_REQUESTED** [review]
Good work.  Just some minor stuff

**Review by brianseidl — APPROVED** [review]

**Pull request merged**